articles/mfa/guides/mfa-api/phone.md

```diff
@@ -186,9 +201,9 @@ If successful, you'll receive the following response, and the user will get an S
 }
 ```
 
-Your application needs to prompt the user for the `binding_code` and send it as part of the request. The `binding_code` is a 6-digit number included in the challenge.
+Your application needs to prompt the user for 6-digit number sent in the message, and should be set in the `binding_code` parameter.
```

Comment:
Should we change `6-digit number ` to `6-digit code` as it might be to close to the phone number?